client/ayon_maya/plugins/publish/collect_farm_instances.py

```diff
@@ -0,0 +1,35 @@
+import pyblish.api
+from ayon_maya.api import plugin
+
+
+class CollectFarmInstances(plugin.MayaInstancePlugin):
+    """Collect Farm Instances for remote publish
+    """
+
+    order = pyblish.api.CollectorOrder + 0.223
+    label = "Collect Farm Instances"
+    families = ["ass", "animation", "pointcache",
+                "oxcache", "proxyAbc", "redshiftproxy",
+                "vrayproxy", "xgen", "yeticache"]
+    targets = ["local"]
+
+    def process(self, instance):
+        if instance.data.get("farm"):
+            instance.data["families"].append("remote_publish_on_farm")
+
+
+class CollectRemoteCacheInstances(plugin.MayaInstancePlugin):
```

Comment:
I think this plugin does not need families filter.

At the end does not have to live in maya, but that would be bigger discussion.